pkg/controller/realmadmin/settings.go

```diff
@@ -66,6 +67,13 @@ func (c *Controller) HandleSettings() http.Handler {
 		TwilioAuthToken    string `form:"twilio_auth_token"`
 		TwilioFromNumber   string `form:"twilio_from_number"`
 
+		Email                bool   `form:"email"`
+		UseSystemEmailConfig bool   `form:"use_system_email_config"`
+		SMTPAccount          string `form:"smtp_account"`
+		SMTPPassword         string `form:"smtp_password"`
```

Comment:
Add a `json:"-"` here to prevent from accidentially logging (zap uses the json serializer)

Reply:
Oops - this isn't important here. It's important on the database model.